docs/man/docker-cp.1.md

```diff
@@ -1,17 +1,50 @@
 % DOCKER(1) Docker User Manuals
 % Docker Community
-% JUNE 2014
+% JANUARY 2015
 # NAME
-docker-cp - Copy files/folders from the PATH to the HOSTPATH
+docker-cp - Copy files/folders between containers or a local path
 
 # SYNOPSIS
 **docker cp**
-CONTAINER:PATH HOSTPATH
+[SRC_CONTAINER:]SRC_PATH [DST_CONTAINER:]DST_PATH
 
 # DESCRIPTION
-Copy files/folders from a container's filesystem to the host
-path. Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+Copy files/folders to or from a container and your local host
+or between any two containers. Container paths are relative to
+the container's working directory. Files can be copied to and
```

Comment:
yea I understand its all possible to do what you showed, I just wonder if its worth the added complexity and possible confusion for people.  Mandating an abs path ( like I think cp requires today ) reduces that risk.